### DISCUSSION TITLE
[Feature] Make machine learning run at specific times

### DISCUSSION THREAD

**Discussion by Sakooooo** [immich-app/immich#2929] [Feature Request]
### The feature

This is such a good app and I've been using it for my photos for awhile but the only complaint i have is machine learning slowing down the server to a halt, is it possible to make machine learning run between times when the server is not being used alot? Thanks!

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#5498
